integration-tests/utils/Common.ts

```diff
@@ -62,4 +67,8 @@ export class Common {
         .should('contain', 'Done running the script');
     }
   }
+
+  static clickBreadcrumbs(breadcrumbPO: string) {
```

Comment:
are you actually using this somewhere?

Reply:
No. Because, currently, there isn't any place in my test where I can use it.
My main intention to use it was [here](https://github.com/openshift/hac-dev/blob/main/integration-tests/utils/Applications.ts#L8), but now since we're using Navigation items to reach the "Applications" page (with fairly nice speed) I don't see a place to fit this.

Might be helpful in future tests, but do we need to pick one from "breadcrumbs" vs "navigation items"?